apps/storybook/src/overlays/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,297 @@
+    disabled: true,
+    id: "book-8",
+    title: "The Brothers Karamazov",
+  },
+  {
+    author: "Lev Tolstoy",
+    disabled: false,
+    id: "book-9",
+    title: "Anna Karenina",
+  },
+  {
+    author: "Fyodor Dostoevsky",
+    disabled: false,
+    id: "book-10",
+    title: "Crime and Punishment",
+  },
+];
+
+const languages = [
+  "English",
+  "French",
+  "German",
+  "Spanish",
+  "Portuguese",
+  "Russian",
+  "Japanese",
+  "Korean",
+  "Chinese (Simplified)",
+  "Italian",
+  "Dutch",
+  "Polish",
+  "Swedish",
+  "Danish",
+  "Finnish",
+  "Norwegian",
+  "Turkish",
+  "Arabic",
+  "Hindi",
+  "Bengali",
+  "Vietnamese",
+  "Thai",
+  "Indonesian",
+  "Greek",
+  "Czech",
+  "Romanian",
+  "Hungarian",
+  "Hebrew",
+  "Ukrainian",
+  "Swahili",
+  "Malay",
+  "Tagalog",
+  "Persian",
+  "Tamil",
+  "Urdu",
+  "Afrikaans",
+  "Bulgarian",
+  "Catalan",
+  "Croatian",
+  "Lithuanian",
+];
+
+const itemToString = (book: unknown) => {
+  return book ? (book as Book).title : "";
+};
+
+const itemToKey = (book: unknown) => {
+  return (book as Book).id;
+};
+
+const isItemDisabled = (book: unknown) => {
+  return (book as Book).disabled;
+};
+
+export const Basic: Story = {
+  render: function Basic() {
+    const [items, setItems] = useState(languages);
+
+    const onInputValueChange = (inputValue: string) => {
+      setItems(
+        languages.filter(
+          (language: string) =>
+            !inputValue ||
+            language.toLowerCase().includes(inputValue.toLowerCase()) ||
+            language.toLowerCase().includes(inputValue.toLowerCase()),
+        ),
+      );
+    };
+
+    return (
+      <Autocomplete items={items} onInputValueChange={onInputValueChange}>
+        <AutocompleteTrigger>
+          <AutocompleteInput placeholder="Search a Language" w="208" />
+        </AutocompleteTrigger>
+        <AutocompleteContent>
+          {items.map((item, index) => (
+            <AutocompleteItem item={item} key={index}>
+              {item}
+            </AutocompleteItem>
+          ))}
+          {items.length === 0 ? (
+            <AutocompleteEmpty>No result Found</AutocompleteEmpty>
+          ) : (
+            <></>
+          )}
+        </AutocompleteContent>
+      </Autocomplete>
+    );
+  },
+};
+
+export const WithLabel: Story = {
+  render: function WithLabel() {
+    const [items, setItems] = useState(books);
+
+    const onInputValueChange = (inputValue: string) => {
+      setItems(
+        books.filter(
+          (book: Book) =>
+            !inputValue ||
+            book.title.includes(inputValue) ||
+            book.author.includes(inputValue),
+        ),
+      );
+    };
+
+    return (
+      <Autocomplete
+        isItemDisabled={isItemDisabled}
+        itemToKey={itemToKey}
+        itemToString={itemToString}
+        items={items}
+        onInputValueChange={onInputValueChange}
+      >
+        <AutocompleteTrigger>
+          <AutocompleteLabel label="Label">
```

Comment:
we don't need this `AutocompleteLabel` component - we can get rid of it entirely